src/main/pages/che-7/end-user-guide/proc_configuring-go-to-use-a-non-standard-registry.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// using-go-artifact-repositories
+
+[id="configuring-go-to-use-a-non-standard-registry_{context}"]
+
+= Configuring Go to use a non-standard-registry
+
+Athens is a Go module datastore and proxy with many configuration options, it can be configured to act only as a module datastore and not as a proxy. An admin can upload their Go modules to the Athens datastore and have them available across their Go projects. If a project tries to access a Go module that is not in the Athens datastore, the Go build fails.
```

Comment:
```suggestion
Athens is a Go module datastore and proxy with many configuration options. It can be configured to act only as a module datastore and not as a proxy. An administrator can upload their Go modules to the Athens datastore and have them available across their Go projects. If a project tries to access a Go module that is not in the Athens datastore, the Go build fails.
```